config/default.yml

```diff
@@ -198,6 +198,12 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', '^<<<<<<<\s']
 
+  Pep257:
```

Comment:
[PEP](https://www.python.org/dev/peps/) is the official channel for Python Enhancement Proposals, which are very much known and followed throughout the Python community.

[PEP 8](https://www.python.org/dev/peps/pep-0008/) and [PEP 257](https://www.python.org/dev/peps/pep-0257) together comprise the official Python [style guide](https://www.python.org/doc/essays/styleguide/) conceived by the language's BDFL, Guido van Rossum.

So yes, I'd wager most Python devs are aware :wink: